NEWS.txt

```diff
@@ -51,11 +69,11 @@ New features
       to be enabled in 4.2; it is controlled with the configuration track_warnings.enabled,
       setting to true will enable this feature.  Each check has its own warn/abort thresholds, currently
       tombstones (tombstone_warn_threshold, and tombstone_failure_threshold), coordinator result set
-      materialized size (track_warnings.coordinator_large_read.warn_threshold_kb, and
-      track_warnings.coordinator_large_read.abort_threshold_kb), local read materialized heap size
-      (track_warnings.local_read_size.warn_threshold_kb and track_warnings.local_read_size.abort_threshold_kb),
-      and RowIndexEntry estimated memory size (track_warnings.row_index_size.warn_threshold_kb and
-      track_warnings.row_index_size.abort_threshold_kb) are supported; more checks will be added over time.
+      materialized size (track_warnings.coordinator_large_read.warn_threshold, and
+      track_warnings.coordinator_large_read.abort_threshold), local read materialized heap size
+      (track_warnings.local_read_size.warn_threshold and track_warnings.local_read_size.abort_threshold),
+      and RowIndexEntry estimated memory size (track_warnings.row_index_size.warn_threshold and
+      track_warnings.row_index_size.abort_threshold) are supported; more checks will be added over time.
     - Prior to this version, the hint system was storing a window of hints as defined by
       configuration property max_hint_window_in_ms, however this window is not persistent across restarts.
```

Comment:
Should we also update `max_hint_window_in_ms` to `max_hint_window` here?

There are some other bits in NEWS.txt still using the old names (ex. `enable_materialized_views`), but it doesn't seem important to change those references as it is for things added in 4.1 (where there isn't actually a previously released name).

Reply:
True, I will update the post 4.0 NEWS.txt when we are done with everything else. Adding a note to myself not to forget. The parameters I corrected are just changed absolutely everywhere as they won't have at all the old version without units because they were introduced in trunk post 4.0 before even alpha release.